### PR TITLE
Update Chrome Extensions.md

### PR DESCRIPTION
**Pull request by ZephrFish**

### CODE REVIEW

File: Chrome Extensions.md [hunk 1]
```diff
@@ -1,30 +1,19 @@
...
+- Bulk URL Opener - https://chrome.google.com/webstore/detail/bulk-url-opener-extension/hgenngnjgfkdggambccohomebieocekm?utm_source=chrome-ntp-icon
+- uBlock Origin - https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?utm_source=chrome-ntp-icon
+- Search by Image - https://chrome.google.com/webstore/detail/search-by-image/cnojnbdhbhnkbcieeekonklommdnndci
+- Go Back in Time - https://chrome.google.com/webstore/detail/go-back-in-time/hgdahcpipmgehmaaankiglanlgljlakj?utm_source=chrome-ntp-icon
+- Stratus Mitre ATT&CK TTPs - https://chrome.google.com/webstore/detail/stratus/jjpceekakjijeamhoeedjfgmhgbefcmp?utm_source=chrome-ntp-icon
+- Link Redirect Trace - https://chrome.google.com/webstore/detail/link-redirect-trace/nnpljppamoaalgkieeciijbcccohlpoh
+- Pulsedive - https://chrome.google.com/webstore/detail/pulsedive-threat-intellig/gdbemlfgncdmmljmkkcemfedbbfgghcp?hl=en
+- Browserling - https://chrome.google.com/webstore/detail/browserling-cross-browser/chfdigebbepdmeelcfiefkklfldmmgld?hl=en
 
 ```
```
Comment (BenMcGarry) [inline Chrome Extensions.md:16]: Is this block required?

**Review by BenMcGarry — COMMENTED** [review]
Is the block at the end needed?

File: Chrome Extensions.md (hunk 1 above)
Comment (ZephrFish) [reply Chrome Extensions.md:16]: I don't think so, just a space

**Review by ZephrFish — COMMENTED** [review]
can confirm works

**Review by BenMcGarry — APPROVED** [review]

**Pull request closed without merging**